pandas/core/internals/managers.py

```diff
@@ -80,10 +81,10 @@
 
 # TODO: flexible with index=None and/or items=None
 
-T = TypeVar("T", bound="BlockManager")
+T = TypeVar("T", bound="_BlockManager")
 
 
-class BlockManager(DataManager):
+class _BlockManager(DataManager):
```

Comment:
can you indicate that this is an abstract class (alt put ABC in the name)

Reply:
good idea

Reply:
updated + green